plugins/sequence/src/IndexedFastaAdapter/IndexedFastaAdapter.ts

```diff
@@ -57,6 +69,10 @@ export default class extends BaseSequenceAdapter {
     }))
   }
 
+  public getHeader(opts?: BaseOptions) {
+    return this.header
```

Comment:
I might consider putting the logic from readHeader directly into getHeader here, and removing the readHeader call from the constructor. this avoids having async stuff called in the constructor which can be hard to error handle

Reply:
I have made it so that we are no longer calling readHeader in the constructor - all work is being done in the getHeader function. This is a lot better as it only does a get request when the user clicks on "About Track". I also made it so that it still stores the results in "this.header". This approach makes it so that it doesn't need to pull the file again if the person goes to "About Track" more than once in a session. Not a huge optimization but still better.